### PR TITLE
feat: Add overrides

### PR DESCRIPTION
**Pull request by nehasri89**
https://nidigitalsolutions.jira.com/browse/REPLAT-5457

The order of priority in headline is tileHeadline || shortHeadline || headline

### CODE REVIEW

**Comment by Ivaylo1987** [comment]
Should there be any tests for those overrides?

File: packages/edition-slices/src/tiles/shared/tile-summary.js [hunk 1]
```diff
@@ -40,7 +41,7 @@ const TileSummary = ({
     flags={() => <ArticleFlags {...flagColour} flags={expirableFlags} />}
     headline={() => (
       <ArticleSummaryHeadline
-        headline={shortHeadline || headline}
+        headline={shortHeadline || tileHeadline || headline}
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/shared/tile-summary.js]: need to test these fallback or coverage will decrease.

Comment (nehasri89) [reply packages/edition-slices/src/tiles/shared/tile-summary.js]: 👍

Comment (tuncaulubilge) [reply packages/edition-slices/src/tiles/shared/tile-summary.js]: This should be `tileHeadline` first

File: packages/edition-slices/src/tiles/shared/tile-tracking-events.js [hunk 2]
```diff
@@ -7,12 +7,15 @@ export default Component =>
...
+          tile: {
+            headline: tileHeadline,
+            article: { headline, id, shortHeadline, url } = {}
+          } = {
             article: {}
           },
           tileName
         }) => ({
-          articleHeadline: shortHeadline || headline,
+          articleHeadline: shortHeadline || tileHeadline || headline,
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/shared/tile-tracking-events.js]: `tileHeadline` first

File: packages/edition-slices/src/tiles/tile-ab/index.js [hunk 3]
```diff
@@ -17,13 +17,13 @@ const TileH = ({ onPress, tile }) => (
         byline={tile.article.byline}
         headlineStyle={styles.headline}
         style={styles.summaryContainer}
-        summary={tile.article.summary125}
+        summary={tile.summary125 || tile.article.summary125}
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/tile-ab/index.js]: It's called `teaser125` on tiles in schema (for some reason)

File: packages/edition-slices/src/tiles/tile-w/index.js [hunk 4]
```diff
@@ -16,14 +16,14 @@ const TileW = ({ onPress, tile }) => (
       <TileSummary
         headlineStyle={styles.headline}
         style={styles.summaryContainer}
-        summary={tile.article.summary125}
+        summary={tile.summary125 || tile.article.summary125}
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/tile-w/index.js]: `teaser125`

File: packages/edition-slices/__tests__/shared-components.base.js [hunk 5]
```diff
@@ -17,14 +18,15 @@ export default () => {
   const tests = [
     {
       name:
-        "Tile summary falls back to headline if shortHeadline is unavailable",
+        "Tile summary falls back to tileHeadline if shortHeadline is unavailable",
```
Comment (tuncaulubilge) [inline packages/edition-slices/__tests__/shared-components.base.js]: Shouldn't be this other way around? Fallback to shortHeadline if tileHeadline is unavailable

File: packages/edition-slices/__tests__/shared-components.base.js [hunk 6]
```diff
@@ -17,14 +18,15 @@ export default () => {
...
-        "Tile summary falls back to headline if shortHeadline is unavailable",
+        "Tile summary falls back to shortHeadline if tileHeadline is unavailable",
       test: () => {
         const tileWithoutShortHeadline = {
           ...tile,
           article: {
             ...tile.article,
-            shortHeadline: ""
-          }
+            shortHeadline: "This is shortheadlin"
```
Comment (ValGeorgiev) [inline packages/edition-slices/__tests__/shared-components.base.js]: A little typo. Depends on you, if you rebase, please fix it

**Pull request merged**